sdk/storage/Azure.Storage.Blobs/swagger/readme.md

```diff
@@ -1044,3 +1044,12 @@ directive:
   transform: >
     $.description = $.description.replace("<GUID>", "{GUID}");
 ```
+
+### Make lease duration a long
+``` yaml

```

Comment:
Please add a descriptive comment here, like the one in your PR, because this is going to make no sense in a couple of months.  :smile: